static/src/deploys-radiator/app/main.ts

```diff
@@ -57,7 +57,7 @@ const run = (): Promise<void> => {
             .sortBy(deploy => deploy.build)
             .first();
 
-        return [ latestCodeDeploy, oldestProdDeploy ];
```

Comment:
I know it is less code but explicitly showing the return type might be handy. No?